code/go/0chain.net/smartcontract/minersc/models.go

```diff
@@ -584,17 +584,17 @@ func (gn *GlobalNode) GetHashBytes() []byte {
 
 // MinerNode struct that holds information about the registering miner.
 type MinerNode struct {
-	*SimpleNode `json:"simple_miner"`
-	Pending     map[string]*sci.DelegatePool `json:"pending,omitempty"`
-	Active      map[string]*sci.DelegatePool `json:"active,omitempty"`
-	Deleting    map[string]*sci.DelegatePool `json:"deleting,omitempty"`
+	*SimpleNode          `json:"simple_miner"`
+	*stakepool.StakePool `json:"stake_pool"`
+	NumPending           int `json:"pending"`
+	NumActive            int `json:"active"`
```

Comment:
These NumPending, NumActive seems are not updated appropriately. The please check all the places that changed the StakePool status, such as MineReward, addToDelegatePool, etc.